capitalize.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 However For the CapitalizeDecorator, you should Implement a method `correct_name` that capitalizes the output of @nameable.correct_name. Here is the actual result 👇🏻 

![Screenshot from 2022-10-12 04-25-33](https://user-images.githubusercontent.com/79058364/195235836-0e77f7a9-491d-4713-9a42-c584d10b1841.png)


- Let's make sure we address this by adding a `capitalize` to the correct_name method, It will be like this:

```ruby
class CapitalizeDecorator < Decorator
    # Capitalize the first letter of the given name.
    def correct_name
      @nameable.correct_name.capitalize
    end
end
```